app/entities/Product.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Entities;
+
+class Product {
+
+    private $name;
+    private $price;
+    private $taxation;
+
+    /**
+     * Constructor
+     *
+     * @param string $name
+     * @param float $price
+     * @param float $taxation
+     */
+    public function __construct(string $name,float $price,float $taxation) {
```

Comment:
Was könnte man tun, um anderen Entwicklern verständlich zu machen, welcher Wert als Steuersatz übergeben werden soll, z.B. `7,7` vs. `0,077` in der Schweiz?